### DISCUSSION TITLE
Error while updating

### DISCUSSION THREAD

**Discussion by sinaomrani96** [idaholab/moose#24136] [Q&A Getting Started] [answered]
Hi everyone.

I wanted to update my moose repository and I am getting the following errors. I wanted to know if it is a general issue or only happened to me. Thanks.

/home/sinaomrani/mambaforge3/envs/moose/libmesh/include/timpi/packing.h:279:22: error: 'pack' is not a member of 'libMesh::Parallel::Packing<MeshInfo::SubdomainInfo, void>'
  279 |     Packing<T3>::pack(comp, data_out, ctx);
      |     ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~

/home/sinaomrani/mambaforge3/envs/moose/libmesh/include/timpi/packing.h:332:31: error: 'unpack' is not a member of 'libMesh::Parallel::Packing<MeshInfo::SubdomainInfo, void>'
  332 |     comp = Packing<T3>::unpack(in, ctx);
      |            ~~~~~~~~~~~~~~~~~~~^~~~~~~~~

/home/sinaomrani/mambaforge3/envs/moose/libmesh/include/timpi/packing.h:234:38: error: 'packable_size' is not a member of 'libMesh::Parallel::Packing<MeshInfo::SidesetInfo, void>'
  234 |     return Packing<T3>::packable_size(comp, ctx);
      |            ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~

and many more.

Best regards,
Sina

**Comment by GiudGiud** [comment, marked as answer]
Hello

You need to update libmesh at the same time as you update moose
`mamba update --all`

Guillaume
